README.md

```diff
@@ -1,26 +1,27 @@
 # puppet_metrics_dashboard
 
-- [Description](#Description)
-- [Setup](#Setup)
-  - [Upgrade notes, breaking changes in v2](#Upgrade-notes-breaking-changes-in-v2)
-  - [Determining where Telegraf runs](#Determining-where-Telegraf-runs)
-  - [Beginning with puppet_metrics_dashboard](#Beginning-with-puppetmetricsdashboard)
-    - [Minimal configuration](#Minimal-configuration)
-- [Usage](#Usage)
-  - [To install example dashboards and use the Telegraf collection method (default)](#To-install-example-dashboards-and-use-the-Telegraf-collection-method-default)
-  - [Configure Telegraf for one or more masters / PuppetDB nodes / Postgres nodes](#Configure-Telegraf-for-one-or-more-masters--PuppetDB-nodes--Postgres-nodes)
-  - [Enable Graphite support](#Enable-Graphite-support)
-  - [Enable Telegraf, Graphite, and Archive (puppet_metrics)](#Enable-Telegraf-Graphite-and-Archive-puppetmetrics)
-  - [Enable SSL](#Enable-SSL)
-  - [Allow access to PE-managed Postgres nodes with the following class](#Allow-access-to-PE-managed-Postgres-nodes-with-the-following-class)
-  - [Profile defined types](#Profile-defined-types)
-    - [Note on using the defined types](#Note-on-using-the-defined-types)
-  - [Other possibilities](#Other-possibilities)
-- [Reference](#Reference)
-- [Limitations](#Limitations)
-  - [Repo failure for InfluxDB packages](#Repo-failure-for-InfluxDB-packages)
-  - [Postgres metrics collection requires Telegraf version 1.9.1 or later](#Postgres-metrics-collection-requires-Telegraf-version-191-or-later)
-- [Development](#Development)
+- [puppet_metrics_dashboard](#puppet_metrics_dashboard)
```

Comment:
I am assuming you started using the VS Code plugin I mentioned to generate this... I suggest editing the settings for the plugin to start with level 2 so that it doesn't add this entry for the title of the page (which is actually above the ToC).